lib/pbench/server/api/resources/query_apis/__init__.py

```diff
@@ -576,9 +576,12 @@ def __init__(
             raise MissingBulkSchemaParameters(
                 api_name, "dataset parameter is not defined or not required"
             )
-        if self.schemas[ApiMethod.POST].authorization != ApiAuthorizationType.DATASET:
+        if self.schemas[ApiMethod.POST].authorization not in [
+            ApiAuthorizationType.ADMIN,
+            ApiAuthorizationType.DATASET,
+        ]:
             raise MissingBulkSchemaParameters(
-                api_name, "schema authorization is not by dataset"
+                api_name, "schema authorization is not by dataset or admin"
```

Comment:
Given Peter's recent changes (on which you'll now need to rebase), we should make this an `assert` that'll trigger on the first build if someone breaks it. There's really no need to detect this at runtime because it's a static code construction problem. Maybe

```
    assert self.schemas[ApiMethod.POST].authorization in [
        ApiAuthorizationType.ADMIN,
        ApiAuthorizationType.DATASET
    ], "Bulk operation authorization type must be ADMIN or DATASET"
```

(Or even `f"API {self.__class__.__name__} authorization type must be ADMIN or DATASET"`)